firedrake/preconditioners/pmg.py

```diff
@@ -262,6 +262,27 @@ def prolongation_transfer_kernel_aij(Pk, P1):
     return kernel
 
 
+def isTensorProductSpace(V):
```

Comment:
This seems misnamed: based on its name I would have expected it to return a single boolean.